modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dynamic/data/mapping/DDMFormFieldUtil.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.headless.delivery.internal.dto.v1_0.util;
```

Comment:
Doing the 2 step (move and adapt) would have moved this changes to the adapt and separate from exposing in the bnd file that could be their own commit because you are changing different things, java and bnds.